CsCore/PlainNetClassLib/PlainNetClassLib.csproj

```diff
@@ -35,11 +35,12 @@
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
```

Comment:
This might be a problem, it has larger implications for Unity applications (where not all environments support this).
Can you remove this line and check the places it is currently needed in your PR if you can rewrite them to be safe?